python_ta/contracts/__init__.py

```diff
@@ -1,7 +1,20 @@
+"""This module provides the functionality for PythonTA contracts.
+
+Representation invariants, preconditions, and postconditions are parsed, compiled, and stored.
+Below are some notes on how they are stored.
+    - Representation invariants are stored in the instance attribute __representation_invariants__
```

Comment:
Don't use the word "instance" here. You can just say "...stored in a class attribute as a list ..."